.rubocop.yml

```diff
@@ -0,0 +1,42 @@
+require:
```

Comment:
Can we use standard for this?
Again, we'd like to do as little yak-shaving as possible and standard has pretty good defaults.